doc/nrf/protocols/bt/bt_mesh/reserved_ids.rst

```diff
@@ -17,59 +17,67 @@ Reserved vendor model IDs
 
 The table below lists all vendor-assigned model identifiers allocated by Nordic Semiconductor:
 
-   +---------------------+-----------------+
-   | Model name          | Vendor Model ID |
+   +-----------------------------+-----------------+
+   | Model name                  | Vendor Model ID |
    +=====================+=================+
```

Comment:
```suggestion
   +-----------------------------+-----------------+
   | Model name                  | Vendor Model ID |
   +=============================+=================+
```
Doc build fails because the table is broken.